hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/ReadableFreezeUpgradeActions.java

```diff
@@ -202,44 +231,57 @@ private void extractAndReplaceArtifacts(
         }
     }
 
-    private void generateConfigPem(@NonNull final Path artifactsLoc) {
+    private void generateConfigPem(@NonNull final Path artifactsLoc, @NonNull final List<ActiveNode> activeNodes) {
```

Comment:
We need to generate config.txt for all nodes in nodeStore, not just ActiveNode. NodeStore and StakingInforStore are not synced in this phase. We only get weight info from StakingInfoStore for active nodes.

Reply:
I could rename `ActiveNode` to `NonDeletedNode`; the only nodes not in this list have `deleted=true`.

What do you think? 🤔

Reply:
I had Streams.stream(nodeIds)
                    .map(nodeId -> nodeStore.get(nodeId.number()))
                    .filter(node -> node != null && !node.deleted())
                    .sorted(Comparator.comparing(Node::nodeId))
                    .forEach(node -> writeConfigLineAndPem(node, bw, artifactsLoc)); before which took care of the delete=true.

Reply:
Ah then the semantics haven't changed. 💯